MapboxCoreNavigation/MMEEventsManager.swift

```diff
@@ -0,0 +1,241 @@
+import Polyline
+import MapboxDirections
+import AVFoundation
+import MapboxMobileEvents
+
+let SecondsForCollectionAfterFeedbackEvent: TimeInterval = 20
+
+struct DefaultEventDictionary {
+    var originalRequestIdentifier: String?
+    var requestIdentifier: String?
+    var lat: CLLocationDegrees?
+    var lng: CLLocationDegrees?
```

Comment:
[Avoid abbreviations](https://swift.org/documentation/api-design-guidelines/#avoid-abbreviations) like `lng` for “longitude”.